<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>790</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame {
	background-color: white;
	border-radius: 20px;
}

#pin_field {
	color: #2681a6;
	font-size: 14pt;
	font-weight: 600;
	border: none;
}

#warning_label {
	color: #d5786c;
}

#terminal {
	margin-right: 80px;
}

#gridFrame .QPushButton {
	width: 70px;
	height: 70px;
	border: none;
	border-radius: 30px;
	color: #2681a6;
	font-size: 14pt;
	font-weight: 600;
}

#gridFrame .QPushButton:hover {
	color: white;
	background-color: #a3c7d6;
}

#gridFrame #clear_btn, #gridFrame #submit_btn {
	background-color: none;
	border: none;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="terminal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>210</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../resources/images/card_insert.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="DraggableCard" name="chip_card">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>180</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../resources/images/chip_card.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="ShadowFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="pin_field">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>INSERT CARD</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="warning_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Incorrect PIN entered</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="gridFrame">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="2" column="1" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn5">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn2">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn4">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn6">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn9">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn8">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn1">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn3">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn7">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="btn0">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="clear_btn">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../resources/icons/delete.svg</normaloff>../../resources/icons/delete.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="submit_btn">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../resources/icons/tick-circle.svg</normaloff>../../resources/icons/tick-circle.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShadowFrame</class>
   <extends>QFrame</extends>
   <header location="global">widgets/shadow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DraggableCard</class>
   <extends>QLabel</extends>
   <header>widgets/draggable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
